### ISSUE TITLE
addreg formula 

### ISSUE THREAD

**Issue by jorgeegm** [biodosetools-team/biodosetools#25]
### Technical details

- Version:
- Assessment:
- Module: fitting with addreg (for a future use)

### Issue description
addreg package dont support formula
ej. ab = -1 + c + alpha + beta
Seems the package dont support formulas with -1 in the formula ...model without intercept not supported...
Without the -1 works find...
The intercept seems to refers to ab value at 0 dose and not to the usual  C term ab/cells

can we obtain the usual C term from the intercept of the model?

**Comment by aldomann** [comment]
This was supported early in the app, but there were some bugs around it. Proper support for models with no intercept is definitely something we want to add in the near future.

**Comment by jorgeegm** [comment]
I agree with intercept . but addreg package do not support the C term in the form ab/cells , like the Braselmann code. 
when I run addreg ( ) then obtain the intercept in the form of ab , e.g. 2.01 and the others parameters alpha and beta as usual , 0.02 and 0.06 because are afected by cells.
